addons/binding/org.openhab.binding.modbus/README.md

```diff
@@ -828,7 +898,7 @@ sitemap modbus_ex_rollershutter label="modbus_ex_rollershutter" {
 
 ```javascript
 // Wrap everything in a function
-// variable "input" contains data passed by openhab
+// variable "input" contains data passed by openHAB
```

Comment:
This documentation could also be made less openHAB specific by referring to openHAB as for instance "the runtime".

Reply:
I agree it could be made more general. On the other hand, perhaps it should be handled outside this PR?